packages/core/src/main/conversation/ConversationService/ConversationService.ts

```diff
@@ -1260,4 +1263,49 @@ export class ConversationService {
       conversation,
     };
   }
+
+  private async sendCommitBundleRemovalMessages(groupIdDecodedFromBase64: Uint8Array, commitBundle?: CommitBundle) {
+    const coreCryptoClient = this.coreCryptoClientProvider();
+
+    if (commitBundle?.welcome) {
+      await this.apiClient.api.conversation.postMlsWelcomeMessage(Uint8Array.from(commitBundle.welcome));
+    }
```

Comment:
Aren’t these welcome & commit already of type uint8array? why do we need to convert?

Reply:
yeah, they are actually!

Reply:
but it doesn't work for some reason, it throws an error: not enough bytes
<img width="523" alt="Screenshot 2022-08-24 at 16 40 38" src="https://user-images.githubusercontent.com/45733298/186447705-c86f39e7-8a1e-4f91-8320-daedb2d29fcc.png">

Reply:
^that one appears on adding user to mls and this one below on removing:
<img width="440" alt="Screenshot 2022-08-24 at 18 05 27" src="https://user-images.githubusercontent.com/45733298/186468017-5920e180-3719-48c8-acfa-6a34f4f38652.png">